core/includes/updates/200-pr12.php

```diff
@@ -157,6 +157,49 @@
     // Continue
 }
 
+// delete language cache since it will contain the old language names and not the short codes
+$cache->setCache('languagecache');
+$cache->eraseAll();
+
+$default_language = $queries->getWhere('languages', ['is_default', '=', 1])[0]->name;
+
+// drop all & truncate from languages table
+try {
+    DB::getInstance()->createQuery('TRUNCATE TABLE nl2_languages');
+} catch (Exception $e) {
+    // Continue
+}
+
+// reset all user languages to default
+try {
+    DB::getInstance()->createQuery('UPDATE nl2_users SET `language_id` = ?', [
+        $default_language[0]->id
```

Comment:
Considering this again, I think we should reset IDs to 1.

If a custom language was set as default, the ID won't exist anymore.

Or, if a custom language was inserted with ID 5, and new languages were added to Nameless with IDs 6+ which were then set as default, the IDs won't match up